service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -88,9 +91,13 @@ public void testGetWithCacheMiss() {
     public void testGetWithCacheHit() {
```

Comment:
Add a test to assert that get returns cached result if eval result is cached but not timestamps for entities.